src/influxdb-reporter.js

```diff
@@ -20,12 +20,15 @@ class InfluxDBReporter {
         failed: [],
         skipped: []
       },
-      list: []
+      list: [],
+      debug: this.reporterOptions.debug
```

Comment:
Maybe move this to start() method, as other --reporter-{...} declaration are specified there..

Reply:
You can have statement like this:
```
    this.context.debug = this.reporterOptions.debug || this.reporterOptions.debug || false;
```

Reply:
changed, but I would leave it like that because I'm already using `debug` in the constructor's if-statement. 
https://github.com/tomaszsluszniak/newman-reporter-influxdb/blob/4ef4537f594c2677fba813bf8a993f3e35e5b738/src/influxdb-reporter.js#L30